packages/dcos-metrics/extra/dcos-metrics.socket

```diff
@@ -0,0 +1,4 @@
+[Unit]
+Description=Metrics Socket: Socket for DC/OS Metrics Service
+[Socket]
+ListenStream=/run/dcos/dcos-metrics.sock
```

Comment:
According to systemd `For each socket file, a matching service file must exist, describing the service to start on incoming traffic on the socket`. which means this needs to have the name `dcos-metrics-master.socket` on masters, `dcos-metrics-agent.socket` on agents I think